Fixes #206

NiteshKant opened the pull request:
Making HttpServerRequest.getContent() and HttpClientResponse.getContent() lazily subscribable.

Commit:
Fixes #206

Making HttpServerRequest.getContent() and HttpClientResponse.getContent() lazily subscribable.

cloudbees-pull-request-builder commented:
[RxNetty-pull-requests #135](https://netflixoss.ci.cloudbees.com/job/RxNetty-pull-requests/135/) FAILURE
Looks like there's a problem with this pull request

Commit:
Merge branch 'master' of https://github.com/Netflix/RxNetty

Commit:
Added missing copyrights

cloudbees-pull-request-builder commented:
[RxNetty-pull-requests #136](https://netflixoss.ci.cloudbees.com/job/RxNetty-pull-requests/136/) FAILURE
Looks like there's a problem with this pull request

NiteshKant commented:
The last failure seems to be because of Cloudbees' local maven repo corruption.

I am disabling local repo for RxNetty builds, that seems to be the only solution for this.

Commit:
Making HttpClient.submit() returning only a single item (as expected)

Also disabling mavenLocal() from the build. This causes cloudbees builds to fail if the local repo gets corrupted.

cloudbees-pull-request-builder commented:
[RxNetty-pull-requests #137](https://netflixoss.ci.cloudbees.com/job/RxNetty-pull-requests/137/) SUCCESS
This pull request looks good

tbak commented on rx-netty/src/main/java/io/reactivex/netty/protocol/http/UnicastContentSubject.java:
```diff
@@ -0,0 +1,354 @@
+
+    private static final IllegalStateException ALREADY_DISPOSED_EXCEPTION =
+            new IllegalStateException("Content stream is already disposed.");
+    public static final IllegalStateException MULTIPLE_SUBSCRIBERS_EXCEPTIONS =
```
It saves CPU/memory to have it pre-created, but stack trace is misleading to users.

NiteshKant replied:
Agreed. Removed the pre-created exceptions.

Commit:
Removing pre-created exception instances from UnicastContentSubject.
